<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOTAL_ThresholdSet</class>
 <widget class="QDialog" name="TOTAL_ThresholdSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>总电流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>curEdit_1</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>最小值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>最大值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="curEdit_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="curMinBtn_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="curMaxBtn_1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改值:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="curMin_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>64.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="curMax_1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>64.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QDoubleSpinBox" name="curMax_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>64.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总电流(L3)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>curEdit_3</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="curEdit_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="curMaxBtn_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="curMinBtn_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="curMin_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>64.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="curMin_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>64.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总电流(L2)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>curEdit_2</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="curEdit_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="curMaxBtn_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="curMax_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>64.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="curMinBtn_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>curMin_2</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>curMaxBtn_1</tabstop>
  <tabstop>curMinBtn_1</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_point</tabstop>
  <tabstop>btn_enter</tabstop>
  <tabstop>curEdit_2</tabstop>
  <tabstop>curEdit_1</tabstop>
  <tabstop>curMinBtn_2</tabstop>
  <tabstop>curMinBtn_3</tabstop>
  <tabstop>curMaxBtn_2</tabstop>
  <tabstop>curMaxBtn_3</tabstop>
  <tabstop>curMin_1</tabstop>
  <tabstop>btn_del</tabstop>
  <tabstop>curMin_3</tabstop>
  <tabstop>curMax_1</tabstop>
  <tabstop>curMax_2</tabstop>
  <tabstop>curMax_3</tabstop>
  <tabstop>curEdit_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
